app/model/RuleContext.scala

```diff
@@ -33,11 +32,15 @@ case class RuleContext(authContext: AuthContext)(implicit hc: HeaderCarrier) {
     userDetailsConnector.getUserDetails(authority.userDetailsLink)
   }
 
-  lazy val activeEnrolments = enrolments.map { enrolmentSeq =>
+  lazy val activeEnrolmentKeys = enrolments.map { enrolmentSeq =>
```

Comment:
I think it is more readable and performance-wise it does not matter as the hard work is done when getting the enrolments from the upstream service which happens once with a lazy val.